examples/vllm/components/args.py

```diff
@@ -28,13 +27,27 @@
 # Only used if you run it manually from the command line
 DEFAULT_ENDPOINT = "dyn://dynamo.backend.generate"
 DEFAULT_MODEL = "Qwen/Qwen3-0.6B"
+DEFAULT_BASE_PORT = 20000
```

Comment:
these are ports for zmq between a known set of workers correct?

why not use let the os assign the ports and write the port values to etcd and use etcd discovery to coordinate them